migration-engine/connectors/sql-migration-connector/src/sql_renderer.rs

```diff
@@ -109,4 +111,16 @@ pub(crate) trait SqlRenderer {
     fn render_create_namespace(&self, _namespace: sql::NamespaceWalker<'_>) -> String {
         unreachable!()
     }
+
+    fn render_create_extension(&self, _create: &CreateExtension, _schema: &SqlSchema) -> Vec<String> {
+        Default::default()
```

Comment:
I'd prefer to panic here, because if that code gets executed it's an error and we want it to fail as loudly as possible. I'll just change that in a new commit.